src/main/java/org/asamk/signal/commands/UpdateContactCommand.java

```diff
@@ -15,6 +15,10 @@ public void attachToSubparser(final Subparser subparser) {
         subparser.addArgument("-n", "--name")
                 .required(true)
                 .help("New contact name");
+        subparser.addArgument("-e", "--expiration")
+                .required(false)
+                .type(int.class)
+                .help("Specify expiration time of messages");
```

Comment:
Please mention the unit here (seconds) and also document the argument in the `man/signal-cli.1.adoc` man page file.